lsp/server/src/utils/orgUtils.ts

```diff
@@ -311,17 +282,15 @@ export class OrgUtils {
         this.objectInfoInMemoCache.clear();
         this.objectInfoPromises.clear();
         this.connection = undefined;
-        if (this.orgName.length > 0) {
-            try {
-                fs.rmSync(this.objectInfoFolderPath(), {
-                    force: true,
-                    recursive: true,
-                    maxRetries: 3
-                });
-            } catch (e) {
-                console.log(e);
-            }
-            this.orgName = '';
+        try {
+            fs.rmSync(this.sfMobileFolder, {
+                force: true,
+                recursive: true,
+                maxRetries: 3
+            });
```

Comment:
What happens when you're logged into multiple orgs?

Reply:
Every time user log into a new org, the cache under '.sfMobile' is cleared. It behaves the same when user switch org

Reply:
Isn't this more destructive? Previous version of the code was specified to remove the folder at object info folder path. Whereas, your code nukes at the `.sfMobile` level. Are we certain that we want to do this?

Reply:
Good question. It is more secure to me. When user logs/switches into a new org or log out, the cached data from previous org should be wiped out to prevent other people peeking into it. We can discuss this topic on the stand-up.

Reply:
If your argument stands then why wasn't it done your way previously?

Reply:
Furthermore. 

Look at my comment on how `ojectInfoFolderPath()` is constructed. Now, I don't remember if anything can get written in `.sfdx` folder in the current setup. If it does, then, isn't it better to keep how you were removing the folders previously than just `sfMobileFolder`?

Also, I vaguely remember if we cache things for the wizard. Could nuking `.sf` folder have any ill effect?

Reply:
Got it. So .sfMobile is only for stashing object info. And watching .sf and .sfdx is for auth status?

Reply:
yes